### DISCUSSION TITLE
Is it possible to have a git shell in this plugin?

### DISCUSSION THREAD

**Discussion by ooker777** [Vinzent03/obsidian-git#348] [General]
I just wonder if it's possible to have a git shell to be able to use any git command?

**Comment by Vinzent03** [comment]
No, there is currently no way to see the executed git commands.

**Reply by ooker777** [reply]
why is that?

**Reply by Vinzent03** [reply]
Actually, I just found out there is. You have to show `verbose` log level in the console. You can open the console via `CTRL + SHIFT + I`.
